Saleel/awsx 588 update gov role based docs (#21494)
DataDog/documentation: content/en/logs/log_configuration/rehydrating.md

## Historical views

> With historical views, teams rehydrate archived log events precisely by timeframe and query filter to meet specific, unexpected use cases efficiently.

Reviewer: ```suggestion
With historical views, teams can rehydrate archived log events by timeframe and query filter for specific use cases that come up unexpectedly.
```

Author: I'm not on the logs team, so i'd prefer not to edit their wording as a driveby change, but we could if you really want
